src/braket/experimental/autoqasm/instructions/qubits.py

```diff
@@ -37,6 +39,26 @@ def is_qubit_identifier_type(qubit: Any) -> bool:
     return isinstance(qubit, QubitIdentifierType.__args__)
 
 
+def physical_qubit_to_braket_qubit(qids: List[str]) -> QubitSet:
```

Comment:
This seems like a weird public method to have. Something like `QubitSet.from_physical_qubits()` might be cleaner.

Reply:
Good point. I'll make this private and move the `QubitSet`-related code back into the `pulse` module. I want to share only the code which parses physical qubit labels (which is likely to be changed soon anyway).